common/pulp/common/constants.py

```diff
@@ -66,6 +66,9 @@
 # The maximum number of seconds that will ever elapse before the scheduler looks for
 # new or changed schedules
 CELERY_MAX_INTERVAL = 90
+# The amount of time the migration script will wait to confirm that no processes are running
+# This is the old CELERY_TICK_DEFAULT_WAIT_TIME + delta, to account for migration scenarios
```

Comment:
Can this be more explicit by version those who are unfamiliar. I think a longer comment is fine. Consider something like:

`Pulp 2.12 lowered the heartbeat time from 90 seconds to 5 seconds. 92 seconds is used to allow the process check feature to work correctly even in cases where a user forgot to restart pulp_celerybeat while upgrading from 2.11 or earlier. The additional 2 seconds allows the database to quiesce so all processes have read consistency.`